testing/test_doctest.py

```diff
@@ -227,7 +227,7 @@ def test_doctest_unex_importerror_with_module(self, testdir):
         # doctest is never executed because of error during hello.py collection
         result.stdout.fnmatch_lines([
             "*ERROR collecting hello.py*",
-            "*ImportError: No module named *asdals*",
+            "*Error: No module named *asdals*",
```

Comment:
I think it might be good to keep the match strict here.
a) It could be changed to cause an ImportError for py36, too.
b) it could be asserted looking at `result.stdout.lines` directly, using a regular expression.
c) there could be an alternative check for py36 and later.

I would prefer c).